gcs/blockcf2/blockcf.go

```diff
@@ -321,9 +322,10 @@ func Regular(block *wire.MsgBlock, prevScripts PrevScripter) (*gcs.FilterV2, err
 	// byte of the script and removing it allows users of the filter to only
 	// match against a normal P2PKH or P2SH script, instead of many extra
 	// matches for each tag.
+	isTreasuryEnabled := false // XXX is it ok to call this with treasury disabled?
```

Comment:
Same here.  Need a `blockcf3` with v3 filters that properly support the new treasury transaction types and are gated behind the flag properly.

Reply:
Per our conversations, we are adding treasury to v2 filters. This change has been committed.